src/lib/libdylink.js

```diff
@@ -1159,9 +1159,9 @@ var LibraryDylink = {
 #if ASYNCIFY
   _dlopen_js__async: true,
 #endif
-  _dlopen_js: (handle) => {
+  _dlopen_js: {{{ asyncIf(ASYNCIFY == 2) }}} (handle) => {
 #if ASYNCIFY
-    return Asyncify.handleSleep((wakeUp) => {
+    return {{{ awaitIf(ASYNCIFY == 2) }}} Asyncify.handleSleep((wakeUp) => {
```

Comment:
if the `await` is being done here why does it also need to be done in in the wrapper generated by jsifier?

Should we not do just one of them?

Reply:
Ah yeah, started with a different approach and this wasn't needed anymore.